content/courses/Layouts.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Layouts
```

Comment:
This should probably say "CSS Layouts" if the focus of this course is writing CSS.